app/src/components/settings/GeneralSettings.tsx

```diff
@@ -18,75 +20,82 @@ const Styled = {
 
 const GeneralSettings: React.FC = () => {
   const { l } = usePrefixedTranslation('cmps.settings.GeneralSettings');
-  const { appView, settingsStore, nodeStore } = useStore();
+  const { appView, log, nodeStore, settingsStore } = useStore();
+  const navigate = useNavigate();
+  const navigateTo = (path: string) => {
+    log.info('Switch to Setting screen', path);
+    navigate(`/settings/${path}`);
+  };
 
-  const handleUnit = useCallback(() => appView.showSettings('unit'), [appView]);
-  const handleBalance = useCallback(() => appView.showSettings('balance'), [appView]);
-  const handleExplorers = useCallback(() => appView.showSettings('explorers'), [appView]);
+  const handleUnit = useCallback(() => navigateTo('unit'), [appView]);
+  const handleBalance = useCallback(() => navigateTo('balance'), [appView]);
+  const handleExplorers = useCallback(() => navigateTo('explorers'), [appView]);
   const handleCopyPubkey = useCallback(() => nodeStore.copy('pubkey'), [nodeStore]);
   const handleCopyAlias = useCallback(() => nodeStore.copy('alias'), [nodeStore]);
   const handleCopyUrl = useCallback(() => nodeStore.copy('url'), [nodeStore]);
 
   const { Wrapper, Content } = Styled;
   return (
-    <Wrapper>
-      <PageHeader title={l('pageTitle')} />
-      <Content>
-        <HeaderFour>{l('general')}</HeaderFour>
-        <SettingItem
-          name={l('bitcoinUnit')}
-          value={formatUnit(settingsStore.unit)}
-          onClick={handleUnit}
-          icon="arrow"
-        />
-        <SettingItem
-          name={l('balances')}
-          value={l('balancesValue', {
-            mode: l(`enums.BalanceMode.${settingsStore.balanceMode}`),
-          })}
-          onClick={handleBalance}
-          icon="arrow"
-        />
-      </Content>
-      <Content>
-        <HeaderFour>{l('explorers')}</HeaderFour>
-        <SettingItem
-          name={l('bitcoinTx')}
-          value={extractDomain(settingsStore.bitcoinTxUrl)}
-          onClick={handleExplorers}
-          icon="arrow"
-        />
-        <SettingItem
-          name={l('lightningNode')}
-          value={extractDomain(settingsStore.lnNodeUrl)}
-          onClick={handleExplorers}
-          icon="arrow"
-        />
-      </Content>
-      <Content>
-        <HeaderFour>{l('myNode')}</HeaderFour>
-        <SettingItem
-          name={l('pubkey')}
-          value={nodeStore.pubkeyLabel}
-          onClick={handleCopyPubkey}
-          icon="copy"
-        />
-        <SettingItem
-          name={l('alias')}
-          value={nodeStore.alias}
-          onClick={handleCopyAlias}
-          icon="copy"
-        />
-        {nodeStore.urlLabel && (
+    <Layout>
```

Comment:
Wrapping this component in `Layout` as only `GeneralSetting` needs the `Hamburger` menu while others show `<- Settings`.

Reply:
I think it would be fine to keep the existing behavior of showing the hamburger/sidebar along with the `<- Settings` link.